index.js

```diff
@@ -0,0 +1,12 @@
+const {server} = require('./Api/server');
+const PORT = 5000;
```

Comment:
consider declaring PORT using this in your dotenv

Reply:
Declared port inside .env.